atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/sweep/metrics/SweepMetricsManager.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2017 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the BSD-3 License (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://opensource.org/licenses/BSD-3-Clause
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.palantir.atlasdb.sweep.metrics;
+
+import com.google.common.annotations.VisibleForTesting;
+import com.palantir.atlasdb.AtlasDbMetricNames;
+import com.palantir.atlasdb.keyvalue.api.SweepResults;
+import com.palantir.atlasdb.keyvalue.api.TableReference;
+
+@SuppressWarnings("checkstyle:FinalClass")
```

Comment:
nit: I'm guessing this is for Mockito? Would be good to leave a comment

Reply:
It was, but since this was not mocked anymore, might as well make it final. If someone desperately wants to extend, we can change it later :)